modules/distribution/product/src/main/resources/conf/infer.json

```diff
@@ -76,9 +74,29 @@
       "database.apim_db.url": "jdbc:mysql://$ref{database.apim_db.hostname}:$ref{database.apim_db.port}/$ref{database.apim_db.name}",
       "database.apim_db.validationQuery": "SELECT 1"
     },
+    "oracle": {
+      "database.apim_db.driver": "oracle.jdbc.OracleDriver",
+      "database.apim_db.url": "jdbc:oracle:thin:@$ref{database.apim_db.hostname}:$ref{database.apim_db.port}/$ref{database.apim_db.sid}",
+      "database.apim_db.validationQuery": "SELECT 1 FROM DUAL"
+    },
+    "mssql": {
+      "database.apim_db.driver": "com.microsoft.sqlserver.jdbc.SQLServerDriver",
+      "database.apim_db.url": "jdbc:sqlserver://$ref{database.apim_db.hostname}:$ref{database.apim_db.port};databaseName\u003d$ref{database.apim_db.name};SendStringParametersAsUnicode\u003dfalse",
```

Comment:
will it break if we replace `\u003d` with `=`?

Reply:
I don't think so.

Reply:
should it be changed to `=`, for readability?

Reply:
yes, please.